Content.Server/Disposal/Unit/EntitySystems/DisposableSystem.cs

```diff
@@ -136,9 +136,9 @@ public void ExitDisposals(EntityUid uid, DisposalHolderComponent? holder = null,
                 {
                     _xformSystem.AttachToGridOrMap(entity, xform);
 
-                    if (holder.PreviousDirection != Direction.Invalid && _xformQuery.TryGetComponent(xform.ParentUid, out var parentXform))
+                    if (holder.PreviousDirection != Direction.Invalid && gridUid != null && _xformQuery.TryGetComponent(gridUid, out var parentXform))  // DeltaV begin EE Disposal fix
```

Comment:
why does this need to use the grids world rotation, the disposal unit should be the same as the grid